src/item.h

```diff
@@ -853,6 +853,12 @@ class item : public visitable<item>
         void reset_temp_check();
 
         int get_comestible_fun() const;
+        /**
+        * Find the containing pocket for this item in the character's inventory.
+        * @param who Character holding the item.
+        * @returns item_pocket Containing pocket. nullptr if not found.
+        */
+        item_pocket &get_containing_pocket( Character &who );
```

Comment:
It returns a reference, not a pointer. Contrary to the description it won't return `nullptr` if nothing is found, it will crash trying to dereference `nullptr`.

Reply:
Will probably remove this method anyway once I look it over again.